modules/ROOT/pages/endpoints.adoc

```diff
@@ -15,7 +15,7 @@ Your organization also has internal endpoints to which it routes messages from t
 * Acknowledgement endpoints +
 Your partners provide you with their own endpoint configurations to which your organization can send acknowledgments, such as an X12 997 or X12 999. Acknowledgements are typically sent back to trading partners and correspond to one or more B2B messages you receive from your trading partners.
 
-Refer to the following to configure your endpoint. Your configuration varies based on the protocol and type you use.
+Refer to the following to configure your endpoint. Your configuration varies based on the protocol and whether you are using send or receive.
```

Comment:
@mcherem the type of an endpoint is not a user selectable/visible field. Configuration screens auto-adjusts based on the context. Having "send or receive" here might confuse it with the direction of message flow. Should we remove it from here and the table below - https://github.com/mulesoft/docs-partner-manager/pull/65/files#diff-4e45fccefae387c7ab46720c13a8e16fR52

Reply:
Fixed.